frontend/src/metabase/public/components/widgets/EmbedCodePane.jsx

```diff
@@ -10,8 +10,6 @@ import {
 } from "../../lib/code";
 import CodeSample from "./CodeSample";
 
-import "metabase/lib/ace/theme-metabase";
```

Comment:
This theme is only used here, and `TextEditor` is only used here, so I put theme styles into `TextEditor`, which is a custom wrapper for ace used here.